client/src/Components/Feature/Feature.tsx

```diff
@@ -3,13 +3,23 @@ import { useFeatureFlags } from '@/Components/FeatureFlagsContext/FeatureFlagsCo
 
 interface FeatureProps {
   flag: string;
+  invert: boolean | string;
```

Comment:
Why not just make a boolean?

Reply:
Get an error when trying to add it in the  <Feature flag="useIQEngineOutReach" invert="false"> without quotes, but with quotes it gets parsed as a string.

Reply:
But with your link below I see how to do it, cheers!